modules/Bio/EnsEMBL/Compara/RunnableDB/OrthologQM/PrepTableCopy.pm

```diff
@@ -95,6 +97,9 @@ sub write_output {
 	} else {
 		@copy_dataflow = @{ $self->_split_into_chunks($self->param('uniq_mlss_list'), $chunk_size, $self->param('compara_db')) };
 	}
+
+    $self->_check_alignments_exist(\@copy_dataflow);
```

Comment:
These changes were not described in the PR overview. All major changes should be included there to make the review process easier. That's the point of the PR template.